packages/core/src/awsService/appBuilder/wizards/serverlessLandWizard.ts

```diff
@@ -0,0 +1,290 @@
+     * @returns Array of pattern objects containing labels and descriptions
+     */
+    public getPatterns(): { label: string; description?: string }[] {
+        if (!this.metadata) {
+            return []
+        }
+        return Object.entries(this.metadata.patterns).map(([patternName, patternData]) => {
+            let description: string | undefined = undefined
+            if (typeof patternData === 'string') {
+                description = patternData
+            } else if (Array.isArray(patternData)) {
+                // If description is an array, join it into a single string
+                description = patternData.join(' ')
+            }
+            if (!patternData || !patternData.name) {
+                return {
+                    label: patternName,
+                    description: description || 'No description available',
+                }
+            }
+            return {
+                label: patternName,
+                description: patternData.description,
+            }
+        })
+    }
+
+    /**
+     * Gets available runtimes for a specific pattern
+     * @param pattern The pattern name to get runtimes for
+     * @returns Array of runtime options with labels
+     */
+    public getRuntimes(pattern: string): { label: string }[] {
+        const patternData = this.metadata?.patterns?.[pattern]
+        if (!patternData || !patternData.runtimes) {
+            return []
+        }
+        return patternData.runtimes.map((runtime) => ({
+            label: runtime.name,
+        }))
+    }
+
+    /**
+     * Gets available Infrastructure as Code options for a specific pattern
+     * @param pattern The pattern name to get IaC options for
+     * @returns Array of IaC options with labels
+     */
+    public getIacOptions(pattern: string): { label: string }[] {
+        const patternData = this.metadata?.patterns?.[pattern]
+        if (!patternData || !patternData.iac) {
+            return []
+        }
+        return patternData.iac.map((iac) => ({
+            label: iac.name,
+        }))
+    }
+}
+
+/**
+ * Wizard for creating Serverless Land applications
+ * Guides users through the project creation process
+ */
+export class CreateServerlessLandWizard extends Wizard<CreateServerlessLandWizardForm> {
+    private metadataManager: MetadataManager
+
+    public constructor(context: { schemaRegions: Region[]; defaultRegion?: string; credentials?: AWS.Credentials }) {
+        super({
+            exitPrompterProvider: createExitPrompter,
+        })
+        this.metadataManager = MetadataManager.getInstance()
+    }
+
+    public override async run(): Promise<CreateServerlessLandWizardForm | undefined> {
+        try {
+            // Load metadata from JSON file
+            const projectRoot = path.resolve(__dirname, '../../../../../')
+            const metadataPath = path.join(projectRoot, 'src', 'awsService', 'appBuilder', 'models', 'metadata.json')
```

Comment:
Does this work when the extension is compiled and loaded through a VSIX or through the official marketplace?

Reply:
I'll check that

Reply:
This is still part of the latest version. Did you try it with the built extension? I think we can change it later, but we need to make sure we test the right case. (I feel there should be another proper way of reading the file instead of just going up directories starting on `__dirname`, but maybe I'm wrong and this is the way)